bigquery/google/cloud/bigquery/job.py

```diff
@@ -2582,8 +2582,8 @@ def to_api_repr(self):
     def _set_properties(self, api_response):
         """Update properties from resource in body of ``api_response``
 
-        :type api_response: dict
-        :param api_response: response returned from an API call
+        Args:
+            api_response (dict): response returned from an API call.
```

Comment:
By the way, the types are specified with names from the `typing` module, thus a `dict` should be named `Dict`, for example. Or `Dict[key_type, value_type]` if you also want to specify the dict content's type(s).

Reply:
got it. The `dict` in question would be a nested API response so it would be okay to just name it `Dict` without specifying the content, right?

Reply:
I suppose so, yes. If all keys are strings, `Dict[str, Any]` could be used, but just `Dict` with a meaningful description is fine, too.